feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetFavoriteSessionsFragment.kt

```diff
@@ -59,7 +59,8 @@ class BottomSheetFavoriteSessionsFragment : DaggerFragment() {
         InjectedViewModelProviders.of(requireActivity()).get(sessionPagesStoreProvider)
     }
 
-    private val groupAdapter = GroupAdapter<ViewHolder<*>>()
+    private val groupAdapter
+        get() = binding.sessionsRecycler.adapter as GroupAdapter<*>
```

Comment:
To suppresses the warning of `Unchecked cast`, use `GroupAdapter <*>` instead of `GroupAdapter<ViewHolder<*>>`.